en/docs/Learn/Tutorials/subscribe-to-an-api.md

```diff
@@ -1,195 +1,201 @@
         ```
 
-        ``` java
-                    curl -k -H "Authorization :Bearer 3dfafa3a-b1e3-3550-8a25-88e4b4fe2fb3" 'https://localhost:8243/phoneverify/1.0.0/CheckPhoneNumber?PhoneNumber=18006785432&LicenseKey=0'
+-  [**Output**](#9c9d5499436444b9ab906c5eff942963)
+        ```xml
+            <?xml version="1.0" encoding="UTF-8"?>
+            <PhoneReturn xmlns="http://ws.cdyne.com/PhoneVerify/query" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+                <Company>Toll Free</Company>
+                <Valid>true</Valid>
+                <Use>Assigned to a code holder for normal use.</Use>
+                <State>TF</State>
+                <RC />
+                <OCN />
+                <OriginalNumber>18006785432</OriginalNumber>
+                <CleanNumber>8006785432</CleanNumber>
+                <SwitchName />
+                <SwitchType />
+                <Country>United States</Country>
+                <CLLI />
+                <PrefixType>Landline</PrefixType>
+                <LATA />
+                <sms>Landline</sms>
+                <Email />
+                <AssignDate />
+                <TelecomCity />
+                <TelecomCounty />
+                <TelecomState>TF</TelecomState>
+                <TelecomZip />
+                <TimeZone />
+                <Lat />
+                <Long />
+                <Wireless>false</Wireless>
+            </PhoneReturn>
         ```
 
-        In the above example, the placeholders mentioned in the cURL command format are replaced as follows:
 
-        -   **&lt;access token&gt;** : Give the test token generated in step 8. Click **Applications** , click on the respective application, which in this case is TestApp, click **Production Key** , and click **copy button** to copy the access token.
+In the above example, the placeholders mentioned in the cURL command format are replaced as follows:
 
-                        !!! warning
-            Make sure you have updated the flash plugin in your web browser in order to get the **copy button** working.
+-   **&lt;access token&gt;** : Give the test token generated in step 8.
+-   **&lt;API URL&gt;** : Click the respective API, in this case, PhoneVerification - 1.0.0. When the API's **Overview** tab appears in the API Store, copy the production URL and append the required query params to it.
```

Comment:
Here as well.

Reply:
fixed in https://github.com/wso2/docs-apim/pull/326/commits/d3856d812d2db0cbb9b24c6374887d904f23ba61